src/components/pages/helper/googlemap.js

```diff
@@ -1,27 +1,108 @@
-import React, { Component } from 'react';
-import { Map, GoogleApiWrapper } from 'google-maps-react';
+import React, {Component } from 'react';
+import { Map, GoogleApiWrapper, Marker } from 'google-maps-react';
+import Pagination from "react-bootstrap/Pagination";
 
 const mapStyles = {
-  width: '100%',
-  height: '100%'
+  width: '100vw',
+  height: '60vh'
 };
 
 export class MapContainer extends Component {
+
+    constructor(props) {
+        super(props);
+        this.state = {
+            error: null,
+            isLoaded: false,
+            items: [
```

Comment:
@tombjarne this is minor, as sample data will change soon, but this data structure could be simplified to avoid the double items data structure.
items: [
  {
     "name": "Freddy..." etc
  }
]

Reply:
@santiagoarriaga sure! I will need to change that for sure with the proper format that will be sent from the backend. Definitely will do that, thanks for pointing it out